docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Keyboard shortcuts
+metaTitle: Keyboard shortcuts - Guide - Handsontable Documentation
+permalink: /next/keyboard-shortcuts
+canonicalUrl: /keyboard-shortcuts
+tags:
+- key bindings
+- keymap
+- shortcut manager
+- keyboard navigation
+---
+
+# Keyboard shortcuts
+
+Use and manage Handsontable's keyboard shortcuts.
+
+[[toc]]
+
+## About keyboard shortcuts
+
+You can intuitively navigate Handsontable with a keyboard, using the out-of-the-box [default keyboard shortcuts](#default-keyboard-shortcuts).
+
+You can also customize the entire set of keyboard shortcuts and handle assigned actions, by:
+- [Removing default keyboard shortcuts](#removing-default-keyboard-shortcuts)
+- [Adding custom keyboard shortcuts](#adding-custom-keyboard-shortcuts)
+- [Replacing default keyboard shortcuts](#replacing-default-keyboard-shortcuts)
+- [Using beforeKeyDown hook](#using-beforekeydown-hook)
+
+## Default keyboard shortcuts
+
+By default, Handsontable features the following keyboard shortcuts:
+
+#### NAVIGATION
+
+| Windows                           | macOS                                                                                        | Action                                       |
+| --------------------------------- | -------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| <kbd>Up↑</kbd>                    | <kbd>Up↑</kbd>                                                                               | Move one cell up                             |
+| <kbd>Down↓</kbd>                  | <kbd>Down↓</kbd>                                                                             | Move one cell down                           |
+| <kbd>Right→</kbd>                 | <kbd>Right→</kbd>                                                                            | Move one cell to the right                   |
+| <kbd>Left←</kbd>                  | <kbd>Left←</kbd>                                                                             | Move one cell to the left                    |
+| <kbd>Tab</kbd>                    | <kbd>Tab</kbd>                                                                               | Move one cell to the right                   |
+| <kbd>Tab</kbd> + <kbd>Shift</kbd> | <kbd>Tab</kbd> + <kbd>Shift</kbd>                                                            | Move one cell to the left                    |
```

Comment:
It is a good occasion to fix the order of modifier keys. It should be <kbd>Shift</kbd> + <kbd>Tab</kbd>

Ref: https://ux.stackexchange.com/questions/58185/normative-ordering-for-modifier-key-combinations